spec/middleman_prismic_spec.rb

```diff
@@ -1,11 +1,45 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe MiddlemanPrismic do
-  it 'has a version number' do
-    expect(MiddlemanPrismic::VERSION).not_to be nil
+  describe "download from prismic" do
+    it "downloads content from prismic and installs it in the appropriate place" do
+      within_temp_directory do |directory|
+        Capybara::Discoball.spin(FakePrismic) do |server|
+          change_directory_and_configure_with_prismic(directory, server)
+          run_middleman_prismic
+
+          filename = Dir.children("data/prismic/demo-tests/").first
+          document = YAML.load_file(
+            File.join("data/prismic/demo-tests/", filename),
+          )
+
+          expect(document.id).to eq "referoo"
+        end
+      end
+    end
+  end
+
+  def within_temp_directory
+    Dir.mktmpdir do |directory|
+      Dir.chdir(directory) do
+        yield(directory)
+      end
+    end
+  end
+
+  def change_directory_and_configure_with_prismic(directory, server)
+    ENV["MM_ROOT"] = directory
+    ENV["TEST"] = "true"
+    File.open("config.rb", "w") do |file|
+      file.write <<-CONFIG
```

Comment:
maybe use a `<<~` heredoc for fancy indentation?

Reply:
Did you know that if you use the language name as a heredoc, GitHub will syntax highlight the contents for you?

```rb
File.open("config.rb", "w") do |file|
  file.write <<~RUBY
    activate :prismic do |f|
      f.api_url = "#{server.url}"
    end
RUBY
end
```

Reply:
Double TIL(DTIL/2TIL)!!